apps/src/templates/curriculumCatalog/no_sections_to_assign_dialog.module.scss

```diff
@@ -0,0 +1,34 @@
+.dialogueContainer {
+  height: 18rem;
+  display: flex;
+  flex-flow: column;
+
+  hr {
+    border: 1px solid #000;
```

Comment:
Will color variables like `$neutral_dark` work here instead of hardcoding?

Reply:
Good call: updated!